src/app/pages/vendor-dashboard/components/vendor-filter/vendor-filter.component.ts

```diff
@@ -1,15 +1,95 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import {COMMA, ENTER, N} from '@angular/cdk/keycodes';
+import { MatChipInputEvent } from '@angular/material/chips';
+import { FormBuilder, FormGroup } from '@angular/forms';
+
+export interface workType{
+  name: string;
+  
+}
+
+
 
 @Component({
   selector: 'app-vendor-filter',
   templateUrl: './vendor-filter.component.html',
   styleUrls: ['./vendor-filter.component.scss']
 })
-export class VendorFilterComponent implements OnInit {
+export class VendorFilterComponent{
+
+  addOnBlur = true;
+  readonly separatorKeysCodes = [ENTER, COMMA] as const;
+  workTypes: workType[] = [];  
+
+  statuses = new Map(
+    [['inContract', false], 
```

Comment:
lets change the names of the statuses to be what was recommended in the html. I can go through after to put this in an enum in the types directory